src/sqlfluff/rules/L028.py

```diff
@@ -138,7 +140,14 @@ def _visit_queries(self, query: Query) -> Iterator[LintResult]:
                     self._fix_inconsistent_to,
                     fixable,
                 )
-        for child in query.children:
+        children = list(query.children)
+        # 'query.children' includes CTEs and "main" queries, but not queries in
+        # the "FROM" list. We want to visit those as well.
+        for a in select_info.table_aliases if select_info else []:
```

Comment:
This additional code is now required because of the `SelectCrawler` improvements. Previously, the behavior of `SelectCrawler` was inconsistent about whether it returned queries in the `FROM` list.